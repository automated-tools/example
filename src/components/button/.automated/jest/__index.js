/* eslint-disable */
import runner from '@automated/plugin-jest';
import * as config from '../../.automated';
import * as component from '../../';
config.__dirpath = 'src/components/button';
runner({ config, component });
