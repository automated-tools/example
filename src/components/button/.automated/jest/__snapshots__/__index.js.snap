// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Src: Components: Button Default 1`] = `
<button
  style={
    Object {
      "fontSize": 20,
    }
  }
>
  hello
</button>
`;
